yarn-project/circuits.js/src/structs/global_variables.ts

```diff
@@ -18,6 +18,8 @@ export class GlobalVariables {
     public version: Fr,
     /** Block number of the L2 block. */
     public blockNumber: Fr,
+    /** Slot number of the L2 block */
+    public slotNumber: Fr,
```

Comment:
Is the meaning of this documented in the protocol spec/yellow paper?

Will there be a need to access it in public? (in other words, does there need to be a new avm opcode?)

Reply:
We have got a spec that is going into more detail on it, but it is similar to what ethereum have. It is not really a something I expect that we expose in the AVM as it is more a block building thing really. Devs should never need to care about it, and they will always be able to compute the value by using the timestamp so no opcoded needed in my mind.

Reply:
Great thanks!